docs/using.rst

```diff
@@ -391,6 +391,10 @@ relevant files can be removed from the system with the ``delete`` subcommand::
 
   certbot delete --cert-name example.com
 
+You can also specify the reason for revocations using the ``reason`` flag::
+
+  certbot revoke --cert-path /etc/letsencrypt/live/CERTNAME/cert.pem --reason keycompromise
+
```

Comment:
I personally think this section flows better if you introduce `--reason` right after the revoke subcommand rather than including it after we start talking about delete. Also, it might be good to list the revocation reasons the user can specify.